test/e2e-smoke-tests.sh

```diff
@@ -44,6 +44,9 @@ kubectl delete --ignore-not-found=true -f ${SERVING_YAML} || fail_test
 wait_until_object_does_not_exist namespaces knative-serving || fail_test
 kubectl delete --ignore-not-found=true -f ${MONITORING_YAML} || fail_test
 wait_until_object_does_not_exist namespaces knative-monitoring || fail_test
+# HACK: sleep an extra 60 seconds, as we also have zipkin installed in istio-system namespace, see
```

Comment:
can you explain to me what condition are we waiting to happen from the extra 60 seconds?

Reply:
I changed to wait for zipkin to be deleted as per your suggestion on Slack. And I created an issue for moving it out of `istio-system` namespace.

Does it make more sense to you?

Reply:
yes, i prefer this change as we don't have to wait any longer than necessary.

Reply:
And more important, it's less brittle. :)